.vscode/extensions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "recommendations": [
+    "christian-kohler.path-intellisense",
```

Comment:
Accidentaly change?

Reply:
No, just decided to add it in while I Was doing other stuff 😅